js/Common.js

```diff
@@ -1138,12 +1138,12 @@ if (!CRM.vars) CRM.vars = {};
       }
     }
     return (deferred || new $.Deferred())
-      .done(function(data) {
+      .then(function(data) {
         // If the server returns an error msg call the error handler
         var status = $.isPlainObject(data) && (data.is_error || data.status === 'error') ? 'error' : 'success';
         handle(status, data);
-      })
-      .fail(function(data) {
+      },
```

Comment:
Ah it seems to be these changes. If I apply the common.js diff alone to a site then I get the hang when creating contacts on the fly.